### ISSUE TITLE
Additional funds for the loki foundation

### ISSUE THREAD

**Issue by Lucifer1903** [loki-project/loki-improvement-proposals#13]
Overview

Recently Simon released an analysis regarding potential changes to emissions related to Pulse implementation. As a side note in this report he felt the need to mention this "I do think there does need to be a further conversation about Foundation funding into the future. Even at $2 per Loki, the Foundation reward would barely cover current annualised costs of the project, so I think a deeper conversation about this is worthwhile".

I understand that The Loki Foundation has fund's to continue for another 3 years at it's current spending rate however I think it's good if we start the deeper conversation about this now. 

Proposal

As mining rewards are to be eliminated with Pulse and discussion around changing the emissions is ongoing, I feel like now is a good time to also discuss the Loki Funding System (LFS) https://docs.loki.network/Governance/LokiFundingSystem/
The removal of mining rewards is the perfect time to allocate a percentage of the block reward to LFS, whether that be 2.5% or 5% I don't know and should probably be discussed. However if LFS is implemented then if the the price of loki isn't high enough for the loki foundation to sustain itself off the 5% governance reward then the loki foundation could submit funding requests through LFS for additional funds.

This obviously needs more discussion and other ideas should be put forward here if anyone has suggestions towards continued funding for the Loki Foundation.

**Comment by lewisphil** [comment]
I think with several planned forks between now and three years from now it not *required* to discuss this now.
I would, however, be in favor of a governance plan with:
   1. an allocation of a default 0 loki to the LFS in every block
   2. the LF can request money and, if approved, this block reward is increased to the approved amount, for the approved time

**Comment by Haafingar** [comment]
I agree, there isn't a need to discuss this right now, but certainly we have to start thinking about funding long into the future. Putting the 2.5% from the LFS still doesn't really address the issue unfortunately. 

I like the idea of the LF being able to dynamically request a set of funds over a set period of time, but this doesnt provide a lot of certainty to either the board or the wider network.

**Comment by Sonofotis** [comment]
When LFS is first implemented we can not expect the SNs of the network to be able to govern the network as well as the Foundation can. 

There will always be a role for the foundation and I think in the future their voting power on where the network goes should be based on what percentage of the network they own. I'm very grateful for the foundation as they are spearheading the projects rapid growth but I will ultimately push for a more decentralised governance model.

I'm very much in favor of giving more power to the SN's and the LFS is the best way to do that. The foundation have their own SN's so they could also vote on their own proposals. 

In the long run, if/when the funding that the Foundation has runs out, I'd hope the network evolves more into a decentralised governance model, where the SN's have a larger say as to where the Network's Loki goes (Considering they are the network).

**Comment by Lucifer1903** [comment]
As a service node operator I think it's good to have part of the governance decentralised through the service nodes but I think they shouldn't have more power than the foundation, at the most it should be 50:50 because the foundation is bound to the principles set in the foundation's constitution, while service node operators will do whatever makes them more money.
